src/exportImage.py

```diff
@@ -140,12 +140,13 @@ def create_geopackage(self, bbox, filename, url, taskid, fullPath, resolution):
             'warpOptions': [thread_count]
         }
         result = gdal.Warp(fullPath, url, **kwargs)
-
         if result:
             result.FlushCache()
             result = None
             return True
-        return False
+        else:
+            self.log.error(f'gdal return empty response for task: "{taskid}"')
```

Comment:
1. set the 'failed' state in DB when gdal returns null 
2. add log when committing to kafka - https://github.com/MapColonies/exporter-worker/blob/master/src/taskHandler.py#L33
3.